modules/vaos/app/controllers/vaos/direct_scheduling_facilities_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require_dependency 'vaos/application_controller'
+
+module VAOS
+  class DirectSchedulingFacilitiesController < ApplicationController
+    def index
+      response = systems_service.get_system_facilities(
+        current_user,
+        facilities_params[:system_id],
+        facilities_params[:parent_code],
+        facilities_params[:type_of_care_id]
```

Comment:
This is mapping to `clinical-service`, right?

Reply:
Yes, we have a [WIP swagger](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/appointments/engineering/api-contract.yml) where we mapped type_of_care_id to clinical-service for this and other endpoints.